### ISSUE TITLE
NEW - SATRE: stakeholder engagement and community buy out

### ISSUE THREAD

**Issue by Davsarper** [alan-turing-institute/data-safe-haven-team#66]
# Goal Title
Ensure that institutions evaluate themselves against SATRE and that the momentum is maintained between funded phases

## Description
At the end of the funded phase of SATRE there was a growing community interest, with institutions and stakeholders affirming they would evaluate themselves against it and contribute feedback.

Without ongoing resources the necessary support to ensure that happens cannot be provided and SATRE may end up not being adopted.

## Definition of Done
There are a number of self evlauaitons completed, there is feedback on the spec repository and, ideally, there is an active WG within the TRE Community continuing to work on it.

## Details
- RACI
  - Accountable team or person: RAM
  - Responsible team or person: RPM, REG
  - Informed and consulted people:
- Estimated effort in total FTE: 0.1 - 0.2

## Task and issue breakdown
Breakdown in specific tasks of maximum a month duration, tasks and issues may be added as necessary
Issues may span across repositories when necessary.

- Engage with stakeholders
- Support evlauaitons
- Upload evaluations
- Record feedback and possible modifications
- Establish WG

## Checklist
- [ ] This Story have been agreed with project members, it tackles prioritised work
  - [ ] If not: Label as ForPrioritisation so it discussed in the next monthly meeting, do not set a status  
- [ ] I have filled in the Team Accountabe field
- [ ] I have included this Story in the agreed upon Milestone, set status as Planned
  - [ ] If not: set status as Backlog, to be Planned via weekly meetings or async discussions
- [ ] The work to be done is likely to span over a month
- [ ] I have broken down the work in monthly issues and added them above
- [ ] I have labeled this issue according to its main project: SATRE, TRESA, CORE DSH (ELA) or other (please agree and create a new label if necessary)
- [ ] If known: sum the total effort (points) of tracked issues and add it on the body
- [ ] Select a Story level of effort in the project fields

**Comment by Davsarper** [comment]
Agreed and approved to work on, @harisood we will need to better define it and evaluate effort (and where it comes from)

**Comment by harisood** [comment]
# Jan planning

This is part of the UK TRE Community (working as part of the SATRE UK TRE Community working group).

**Comment by Davsarper** [comment]
February: still part of UK TRE community but dependent on finalising some other work there, in particular website so we can then point to all WGs. Therefore will pick up in March and focus on getting a couple evaluations up in the repo.

**Comment by Davsarper** [comment]
We have been doing the enabling work as aprt of the community to good success, and SATRE is officially a TRE Comm WG but we have had no capacity to do the active part of this that we said we would (actively engage stakeholders, push for self-assessments).

Propose to close or re-consider what the work within it entails

**Comment by Davsarper** [comment]
@jemrobinson a co-chair of the SATRE working group, DSH time for James in June will probably be used up on codebase so not a priority this month but soon it should
